### ISSUE TITLE
Multiple timers on the same project with different tasks show the same timer

### ISSUE THREAD

**Issue by HughMacdonald** [eXpl0it3r/streamdeck-clockify#23]
If you have multiple tasks in Clockify in a project, and have separate StreamDeck buttons linked to each one, when you start a timer, all of the buttons show the same timer.

Looking at the code (https://github.com/eXpl0it3r/streamdeck-clockify/blob/b98db7b354131463ba2c166c6a52813f63b115cc/Clockify/ClockifyContext.cs#L139), it doesn't appear to use the task name when identifying the timer that is currently running.

**Comment by eXpl0it3r** [comment]
Good point, if we do have the task set, we should also filter for it.  
Maybe that was broken in the last refactoring.

**Comment by HughMacdonald** [comment]
I've now hit the point where I actually need to be able to do this - I'd forgotten about this bug. I'm going to see if I can fix it today

**Comment by HughMacdonald** [comment]
That worked for me

That'll be a good workaround for now if I don't figure out the CLI to do the build locally

**Comment by HughMacdonald** [comment]
I thought you were just sending that to test if the build worked, but it's actually fixed up the task filtering issue too - thanks for that - very helpful

**Comment by eXpl0it3r** [comment]
Woops, yeah linked the wrong issue in the PR 😅 

Fixed in #27

Sorry for not working on this sooner. I kind of forgot and been focused on other projects 😓
